analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ParameterLoaderTest.cs

```diff
@@ -119,71 +123,71 @@ public void SetParameterValues_WhenGivenValidSonarLintFileAndDoesNotContainAnaly
         public void SetParameterValues_WithNonExistentPath_UsesInMemoryText()
```

Comment:
What is this test testing? If I understand correctly, the file is written to disk and you use that path to load the rules. So it is not a non existent path. To actually test it, a non existent path should be provided to the `CreateCompilationWithOption` call.

Reply:
That's a good question, I'm not sure exactly. `SetParameterValues_WhenNoSonarLintIsGiven_DoesNotPopulateParameters` checks if default values are read when the SonarLintFile is not provided or the path is nonexistent. With the new implementation, I guess this test is unnecessary, the correct behavior is already fully tested with the test described above and `SetParameterValues_WhenGivenValidSonarLintFilePath_PopulatesProperties`. Wdyt?